### PR TITLE
bump(deps): bump regclient from 0.5.1 to 0.5.6

### PR DESCRIPTION
**Pull request by yeikel**
Release notes : 

https://github.com/regclient/regclient/releases/tag/v0.5.2
https://github.com/regclient/regclient/releases/tag/v0.5.3
https://github.com/regclient/regclient/releases/tag/v0.5.4
https://github.com/regclient/regclient/releases/tag/v0.5.5
https://github.com/regclient/regclient/releases/tag/v0.5.6

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -2,11 +2,13 @@ FROM ghcr.io/dependabot/dependabot-updater-core
 ARG TARGETARCH
 
 # Install regctl. See https://github.com/regclient/regclient/releases for updates
-ARG REGCTL_VERSION=0.5.1
+ARG REGCTL_VERSION=0.5.3
 
 # These are manually calculated as they are not provided by the maintainer
-ARG REGCTL_AMD64_CHECKSUM=484f98e99c90341336b1be99b2b333e7129fde3535b54ef97b4673e8fb2b1d6c
-ARG REGCTL_ARM64_CHECKSUM=17312dadc43ed24886076b0669bc36a9d790c26140fbacb66b0b89f5f96ece52
+# curl -sSf -o regctl-amd64 -L https://github.com/regclient/regclient/releases/download/v${REGCTL_VERSION}/regctl-linux-amd64 && sha256sum regctl-amd64 && rm regctl-amd64
```
Comment (JamieMagee) [inline docker/Dockerfile:8]:
```suggestion
# curl -sL https://github.com/regclient/regclient/releases/download/v${REGCTL_VERSION}/regctl-linux-amd64 | sha256sum
```

**Review by JamieMagee — APPROVED** [review]

File: docker/Dockerfile (hunk 1 above)
Comment (yeikel) [reply docker/Dockerfile:8]: Thank you for the feedback!

**Review by deivid-rodriguez — APPROVED** [review]
I think this should be fine.

Do you have a sample public docker PR where GitHub includes changelog/release info? I'd like to verify there's no obvious regressions against a real repo.

**Comment by yeikel** [comment]
> Do you have a sample public docker PR where GitHub includes changelog/release info? I'd like to verify there's no obvious regressions against a real repo.

Do you mean something like these? 

https://github.com/radiorabe/container-image-minio/pull/21
https://github.com/dronestock/mcu/pull/70/files

I discovered that example searching for [author:app/dependabot is:pr label:docker "release notes" ](https://github.com/pulls?q=author%3Aapp%2Fdependabot+is%3Apr+label%3Adocker+%22release+notes%22)

**Comment by deivid-rodriguez** [comment]
Yeah, exactly what I meant, thanks also for sharing your idea to find these 👍.

**Comment by jeffwidman** [comment]
This looks ready to merge, anything holding it back?

**Comment by yeikel** [comment]
> This looks ready to merge. Is there anything holding it back?

Not that I am aware of, but @deivid-rodriguez can confirm

Ps. I'm happy to see you back! 🥳

**Comment by yeikel** [comment]
@deivid-rodriguez `0.5.6` was just released and I updated this PR. if you prefer to release 0.5.5 first, we can revert 803d8b200fdaf7219d2617c9bd038b130313bc76

**Comment by yeikel** [comment]
@deivid-rodriguez Friendly ping. This seems good to go according to the CI

**Review by abdulapopoola — APPROVED** [review]

**Pull request merged**